### PR TITLE
v0.8.3

### PR DESCRIPTION
**Pull request by AurevoirXavier**

### CODE REVIEW

**Comment by AurevoirXavier** [comment]
Waiting https://github.com/darwinia-network/darwinia-common/pull/389

**Comment by HackFisher** [comment]
@AurevoirXavier Runtime Migration Required:

https://github.com/darwinia-network/darwinia-common/pull/398#pullrequestreview-549647221

**Comment by AurevoirXavier** [comment]
@wuminzhe Pls check your account detail https://github.com/darwinia-network/darwinia/pull/588/commits/8c0a2490d6afe2b9a4251f3377b43950366bb33d

File: runtime/darwinia/src/lib.rs [hunk 1]
```diff
@@ -884,8 +919,15 @@ impl darwinia_ethereum_relay::Trait for Runtime {
 }
 
 type EthereumRelayerGameInstance = darwinia_relayer_game::Instance0;
+parameter_types! {
+	// TODO: migration
```
Comment (HackFisher) [inline runtime/darwinia/src/lib.rs:923]: Do we need this comment any more or can be removed?

File: runtime/crab/src/lib.rs [hunk 2]
```diff
@@ -916,8 +952,15 @@ impl darwinia_ethereum_relay::Trait for Runtime {
 }
 
 type EthereumRelayerGameInstance = darwinia_relayer_game::Instance0;
+parameter_types! {
+	// TODO: migration
+	// pub const EthereumRelayerGameLockId: LockIdentifier = *b"ethrgame";
+	// Workaround
```
Comment (HackFisher) [inline runtime/crab/src/lib.rs:958]: Do we need this comment any more or can be removed?

**Review by HackFisher — APPROVED** [review]

**Comment by HackFisher** [comment]
@AurevoirXavier Ready for merge

**Comment by AurevoirXavier** [comment]
> @AurevoirXavier Ready for merge

Working on adding a sudo to reset authorities.

**Pull request merged**